### PR TITLE
Launcher3: Update Chinese translations

### PR DESCRIPTION
**Pull request by kaffuchino233**

### CODE REVIEW

**Comment by JohnsonRan** [comment]
@jhonboy121 yea,you can merge this

File: res/values-zh-rCN/flamingo_strings.xml [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+-->
+
+<resources>
+    <!-- Folder titles -->
+    <string name="google_folder_title" translatable="false">Google</string>
```
Comment (jhonboy121) [inline res/values-zh-rCN/flamingo_strings.xml]: Do not translate strings marked as translatable="false"

File: res/values-zh-rCN/flamingo_strings.xml [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    <string name="settings_lock_layout_summary_off">����������Ļ����ӡ�ɾ����ƶ�ͼ���С����</string>
+    <string name="settings_edit_widgets_error">�޷���С������ӵ�����Ļ</string>
+
+    <!-- Hide labels -->
+    <string name="desktop_show_labels">����������ʾͼ���ǩ</string>
+    <string name="drawer_show_labels">�ڳ�������ʾͼ���ǩ</string>
+
+    <!-- AppInfo -->
+    <string name="app_info_subtext">���»����Թر�Ӧ����Ϣ��</string>
+    <string name="app_info_version_value"  translatable="false">%1$s (%2$d)</string>
```
Comment (jhonboy121) [inline res/values-zh-rCN/flamingo_strings.xml]: Same here and do fix the others as well.

**Review by jhonboy121 — CHANGES_REQUESTED** [review]
I have commented on specific lines for required changes

**Comment by chingsky-liu** [comment]
> 
I have restored all statements with translatable="false" and force push.

**Comment by jhonboy121** [comment]
> > I restored all statements with translatable="false".

It is still there though, verify you force pushed right. Also force push is not necessary since I am eventually gonna squash and merge all changes in your pr.

**Comment by kaffuchino233** [comment]
Please forgive me for not understanding what you meant earlier,now it's ok.

**Comment by kaffuchino233** [comment]
image
This is a error caused by the editor parsing,I have re-edited and confirmed that the build can be completed.
If do not re-edit, will be prompted with Error parsing XML: not well-formed (invalid token)

**Pull request merged**